### DISCUSSION TITLE
How to use a domain?

### DISCUSSION THREAD

**Discussion by ecvhaoo** [AdguardTeam/AdGuardHome#3016] [Q&A]
Hello 
I am building and using AdGuard Home on Vultr VPS.
Nothing else, there is a private DNS in Samsung Galaxy, and I want to make this, so I ask you a question.

1. The current ip address is entered and used in the dns setting window. What are some ways to enable DNS to be used by entering the domain (dns.example.com)?

2. I have already entered the Vultr IP address in the domain I have in a record and entered it in the private DNS, but it cannot be used normally. What should I do in this case?

**Comment by ameshkov** [comment]
> The current ip address is entered and used in the dns setting window. What are some ways to enable DNS to be used by entering the domain (dns.example.com)?

You need to actually acquire a domain name and then you'll have a DNS panel from your registrar where you'll be able to configure your domain name to point to your Vultr VPS IP address. GoDaddy has quite a few instructions on how to do this.

> I have already entered the Vultr IP address in the domain I have in a record and entered it in the private DNS, but it cannot be used normally. What should I do in this case?

You need to have Encryption configuration in AdGuard Home and for that you also need a valid domain name.

https://github.com/AdguardTeam/AdGuardHome/wiki/Encryption
